### DISCUSSION TITLE
Reports GetAsync() differences between Microsoft.Graph and Microsoft.Graph.Beta

### DISCUSSION THREAD

**Discussion by christophwille** [microsoftgraph/msgraph-sdk-dotnet#1931] [Q&A]
Microsoft.Graph" Version="5.11.0"
Microsoft.Graph.Beta" Version="5.31.0-preview"

For the release version, GetAsync() gives me Stream. For Beta, I get an already properly deserialized List (tried with GetOffice365ActiveUserDetailWithPeriod). So this means .Reports.* is in for a hefty breaking change to the C# API?

**Comment by andrueastman** [comment]
This is not necessarily the case as the beta API may change or the API onboarded with a different path/name on V1 endpoint. 
Ideally, we do not expect breaking changes on V1.

The reports API had some changes sometime back that may explain this inconsistency though. 

https://devblogs.microsoft.com/microsoft365dev/changes-to-microsoft-365-apps-usage-reports-api-in-microsoft-graph/
